fdbclient/ActorLineageProfiler.cpp

```diff
@@ -370,6 +359,17 @@ std::map<std::string, std::string> ProfilerConfigT::getConfig() const {
 	return res;
 }
 
+// Callback used to update the sampling profilers run frequency whenever the
+// frequency changes.
+void samplingProfilerUpdateFrequency(std::optional<std::any> freq) {
```

Comment:
What's the change here? Was this only moved?

Reply:
Yes, just moved it to be next to the other function which watches for a change in the window key.